src/content/getting-started/provision-your-first-workload-cluster/_index.md

```diff
@@ -127,95 +129,45 @@ kubectl gs template cluster \
-
-To _actually_ create the resources–the workload cluster and (only for vintage product family) worker node pool–you need to apply the manifests. Ensure you are still pointing to the management cluster's kubectl context and run:
+To _actually_ create the resources you need to apply the manifests. Ensure you are still pointing to the management cluster's kubectl context and run:
 
 ```sh
 kubectl apply -f cluster.yaml
-
-# Only for vintage product family:
-kubectl apply -f nodepool.yaml
-```
-
-### Deleting the workload cluster {#deleting-workload-cluster}
-
-Deletion works in the same way: run `kubectl delete -f FILENAME.yaml` and the operators in the management cluster will delete the resources in a few minutes. Please don't directly delete the CAPI custom resources (such as `Cluster`, `AWSCluster` or `MachineDeployment`) since this may leave resources behind or even lead to inadvertently recreating the cluster once the `App` is reconciled again. Deletion should be done exactly like the creation, using the original manifests. For the CAPI product family, our example output file `cluster.yaml` contains 2 `App` and 2 `ConfigMap` manifests. If you no longer have the manifests at hand, delete the following:
-
-- `App/<cluster>`
-- `ConfigMap/<cluster>-userconfig`
-
-If you would like to protect your clusters from accidental deletion, take a look at our [deletion prevention mechanism]({{< relref "/vintage/advanced/app-platform/deletion-prevention" >}}).
-
-### Private workload clusters
-
-By default, the created Kubernetes cluster API endpoint is public. See [Private clusters]({{< relref "/vintage/advanced/cluster-management/private-clusters" >}}) if you want to limit networking to/from the cluster.
-
-## Step 4: Watch the status of workload clusters
-
-In the Web UI, click on the cluster's name to see its details. The workload cluster's name is shown on the top-left (in the screenshot: `o8r3r`).
-
-![Adding a nodepool](nodepool.png)
-
-For the vintage product family, if you didn't yet attach a node pool to the WC as shown above, you can now add one by clicking the "Add node pool" button.
-
-Using the command line, you can also watch the creation and status of the workload cluster:
-
-{{< tabs >}}
-{{< tab id="status-vintage" for-impl="vintage_any">}}
-
-```sh
-kubectl gs get clusters -A
-
-kubectl gs get nodepools -A
 ```
 
-{{< /tab >}}
-{{< tab id="status-capi" for-impl="capi_any">}}
+## Step 2: Watch the status of the workload cluster
 
-Either use kubectl-gs:
+You can watch the creation and status of the workload cluster running:
 
 ```sh
 kubectl gs get clusters -A
```

Comment:
What benefit do we get using `kubectl-gs` here? Wouldn't be simpler just use `kubectl get clusters.cluster.x-k8s.io`?

Reply:
The output is enhanced
```
» kgs get clusters -A  
NAMESPACE        NAME    AGE    CONDITION   RELEASE   SERVICE PRIORITY   ORGANIZATION   DESCRIPTION
org-test       api01   52d    READY       25.0.0    medium             Giantswarm         API 1 (testing)
------------------------------------------------------------------------
» k get clusters.cluster.x-k8s.io -A                 
NAMESPACE        NAME    CLUSTERCLASS   PHASE         AGE    VERSION
org-test       api01                  Provisioned   52d

```